packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -66,6 +66,27 @@
                                     data-ng-bind="(flavor.disk | cucBytes:2:false:'GB') + ' ' + (flavor.diskType | uppercase)"
                                 ></span>
                             </div>
+                            <div
+                                class="d-flex justify-content-between"
+                                data-ng-if="flavor.getNvme()"
+                            >
+                                <span
+                                    data-translate="pci_project_flavors_spec_nvme"
+                                ></span>
+                                <span>
+                                    <span
+                                        data-ng-if="flavor.getNvme().disks[0].number > 1"
```

Comment:
This could probably be in the `Flavor` too, something like `getNvmeDiskCount`

Reply:
This is done :)